cheat.php

```diff
@@ -78,8 +78,16 @@
 
 if( ini_get( 'precision' ) < 18 )
 {
-	Msg( '{teal}Fixed php float precision (was ' . ini_get( 'precision' ) . ')' );
+	Msg( '>> {teal}Fixing php float precision (currently ' . ini_get( 'precision' ) . ')' );
 	ini_set( 'precision', '18' );
```

Comment:
ini_set() returns string or bool false on failure, you could write `if ( ini_set( ... ) === false ) { // fail } else { // success }` instead.